src/snmalloc/pal/pal_linux.h

```diff
@@ -29,8 +29,9 @@ namespace snmalloc
      */
     static constexpr uint64_t pal_features = PALPOSIX::pal_features | Entropy;
 
-    static constexpr size_t page_size =
-      Aal::aal_name == PowerPC ? 0x10000 : PALPOSIX::page_size;
+    static constexpr size_t page_size = Aal::aal_name == PowerPC ?
+      0x10000 :
+      (Aal::aal_name == LoongArch ? 0x4000 : PALPOSIX::page_size);
```

Comment:
I wonder if we should add an aal feature for non-4KiB-minimum-page-sizes and, if it's present, expose this as a size in the AAL.  We could then have an `aal_minimum_page_size()` that returned 4KiB if you didn't set this feature and whatever size the AAL required if you did.